elevate-cpanel

```diff
@@ -1952,21 +1991,21 @@ sub _fetch_imunify_installer ($product) {
         return;
     };
 
-    return _fetch_a_script( $installer_url, 'imunify_installer' );
+    return _fetch_a_script( $installer_url, 'imunify_installer', '.sh' );
 }
 
-sub _fetch_a_script ( $url, $template ) {
+sub _fetch_a_script ( $url, $template, $suffix = '' ) {
```

Comment:
why not default suffix to .sh since that's what every existing caller used?

Reply:
I was going off of URLs, and this didn't seem to warrant a `.sh`, even though it got one:

```perl
      my $installer_script = _fetch_a_script( 'https://kernelcare.com/installer', 'kernelcare_installer' );
```

However, on doing an HTTP request, I see that URL redirects to something that does end in `.sh`:

```
# curl -LI https://kernelcare.com/installer
HTTP/1.1 301 Moved Permanently
Date: Thu, 06 Oct 2022 23:42:43 GMT
Content-Type: text/html
Connection: keep-alive
Location: https://repo.cloudlinux.com/kernelcare/kernelcare_install.sh
```